packages/models/src/order/getAllOrderQuery.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface GetAllOrderQuery {
+    shippingStatus?: string;
+    orderStatus?: string;
+    paymentStatus?: string;
+    skip?: number;
+    limit?: number;
+    startDate?: Date;
+    endDate?: Date;
+    startTime?: string;
+    endTime?: string
```

Comment:
front end will send starttime with startDate. actually full date-time can be taken as a single input value. no need to take start time and end time as different field.